src/main/java/org/opensearch/security/securityconf/DynamicConfigFactory.java

```diff
@@ -137,12 +138,14 @@ public DynamicConfigFactory(
         final Path configPath,
         Client client,
         ThreadPool threadPool,
-        ClusterInfoHolder cih
+        ClusterInfoHolder cih,
```

Comment:
Would it make sense to add `clusterName` into `ClusterInfoHolder`? Based on the name of the class it looks like it is a repository for cluster info.

Its only instantiated once in `OpenSearchSecurityPlugin.createComponents(...)` here: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java#L948

Reply:
Fixed.